### PR TITLE
action-build-reporter - Disable main repo at the job level

### PR DESCRIPTION
**Pull request by gsmet**
Just downloading the artifacts takes time so let's disable the run for
the main repository at the job level so nothing is executed.

### CODE REVIEW

**Review by geoand — APPROVED** [review]

**Comment by gsmet** [comment]
Let's wait for the full CI run in my fork and the main repo before merging as I need to compare the behaviors.

**Pull request merged**
